charms/jupyter-ui/src/spawner_ui_config.yaml

```diff
@@ -17,23 +17,23 @@
 spawnerFormDefaults:
   image:
     # The container Image for the user's Jupyter Notebook
-    value: public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/jupyter-scipy:v1.4
+    value: kubeflownotebookswg/jupyter-scipy:v1.5.0
```

Comment:
Is this intentionally v1.5.0?

Reply:
Yes, that's what's in [kubeflow/manifests](https://github.com/kubeflow/manifests/blob/v1.6.0-rc.0/apps/jupyter/jupyter-web-app/upstream/base/configs/spawner_ui_config.yaml) and in [kubeflow/kubeflow](https://github.com/kubeflow/kubeflow/blob/v1.6.0-rc.0/components/crud-web-apps/jupyter/manifests/base/configs/spawner_ui_config.yaml)